src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -911,8 +911,8 @@ If you encounter problems with configuring your log forwarder, try these trouble
 
     * For `HTTPS`, you can configure it in one of the following ways:
 
-    * Import the proxy certificate to the system pool (Recommended)
-      Import the proxy self-signed certificate (PEM file) by using the MMC tool. Refer to [this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), and in **Step 2** ensure to import it in your `Trusted Root Certification Authorities`, instead of in the `Intermediate Certification Authorities`.
+    *  (Recommended) Import the proxy certificate to the system pool
+    * Import the proxy self-signed certificate (PEM file) by using the MMC tool. Refer to [this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), and in **Step 2** ensure to import it in your `Trusted Root Certification Authorities` instead of in the `Intermediate Certification Authorities`.
```

Comment:
```suggestion
    *  (Recommended) Import the proxy certificate to the system pool. Import the proxy self-signed certificate (PEM file) by using the MMC tool. Refer to [this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), and in **Step 2** ensure to import it in your `Trusted Root Certification Authorities` instead of in the `Intermediate Certification Authorities`.
```